protocol/app/ante/sigverify.go

```diff
@@ -34,27 +33,6 @@ func NewSigVerificationDecorator(
 	}
 }
 
-// OnlyLegacyAminoSigners checks SignatureData to see if all
-// signers are using SIGN_MODE_LEGACY_AMINO_JSON. If this is the case
-// then the corresponding SignatureV2 struct will not have account sequence
-// explicitly set, and we should skip the explicit verification of sig.Sequence
-// in the SigVerificationDecorator's AnteHandler function.
-func OnlyLegacyAminoSigners(sigData signing.SignatureData) bool {
```

Comment:
To double confirm, we are deleting this because we can directly use CosmosSDK's method?

Reply:
Yes